fitbenchmarking/utils/fitbm_result.py

```diff
@@ -179,6 +179,24 @@ def __eq__(self, other):
                 return False
         return True
 
+    def get_n_parameters(self):
+        """
+        Returns number of parameters for the result.
+
+        :return: String for number of parameters
+        :rtype: str
```

Comment:
As written these return ints
```suggestion
        :return: Number of parameters
        :rtype: Int
```